frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/multiple-storageclass-selection.spec.ts

```diff
@@ -37,7 +37,6 @@ const addCapacity = (uid: string, scName: string) => {
     .click();
 
   // wait for Pvs & Nodes data to load (used for validations)
-  // to-do (Sanjal): add a more robust way instead of wait by checking the currentCapacity/PVsAvailableCapacity components state
   // eslint-disable-next-line cypress/no-unnecessary-waiting
   cy.wait(5000);
```

Comment:
I think that we have to wait for a few seconds here, so that "pvData" and "nodesData" get loaded (used for validation in `isValidTopology` function in `install.ts` file), then we can click on the “add” button. Otherwise, the test will fail because of validation errors.
I tried this here:
```
cy.byTestID('ceph-add-capacity-element')
  .should(‘be.visible’)
  .byTestID('confirm-action')
  .click();
```
But, this might introduce flakes, cause element with ID 'ceph-add-capacity-element' (i.e., `currentCapacity/PVsAvailableCapacity` components in `add-capacity-modal`) is only dependent on "pvData" not on "nodesData", and we have to wait till both pvData and nodesData gets loaded or else validation will fail and clicking on “add” button will not allow addition of new capacity, hence test will fail.

Reply:
so, in short...
currentCapacity/PVsAvailableCapacity --> these components' state is only dependent on "pvData", thus, we cannot check for their visibility...
cause we only want to click "add" button once both "pvData" and "nodesData" gets loaded...

Reply:
This can be a problem in production too then, lets disable the add capacity button if there are any errors or data is not completely loaded.